modules/slider/Slider.story.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
+import {
+  SliderStory,
+  DisabledSliderStory,
+  LockedSliderStory
+} from "./Slider.story.tsx";
+
+<Meta title="Components | Slider" />
+
+# Slider
+
+## Simple Slider
+
+<Preview>
+  <Story name="Simple Slider">
+    <SliderStory />
+  </Story>
+</Preview>
+
+## Disabled Slider
+
+<Preview>
+  <Story name="Disabled Slider">
+    <DisabledSliderStory />
+  </Story>
+</Preview>
+
+## Locked Slider
```

Comment:
Don't forget to add the available props and extendable styles for both `Slider` and `LockedSlider` 🙂